sys/arm/broadcom/bcm2835/bcm2835_gpio.c

```diff
@@ -770,24 +809,28 @@ bcm_gpio_attach(device_t dev)
 	}
 	sc->sc_bst = rman_get_bustag(sc->sc_res[0]);
 	sc->sc_bsh = rman_get_bushandle(sc->sc_res[0]);
-	/* Setup the GPIO interrupt handler. */
-	if (bcm_gpio_intr_attach(dev)) {
-		device_printf(dev, "unable to setup the gpio irq handler\n");
-		goto fail;
-	}
 	/* Find our node. */
 	gpio = ofw_bus_get_node(sc->sc_dev);
 	if (!OF_hasprop(gpio, "gpio-controller"))
 		/* Node is not a GPIO controller. */
 		goto fail;
+	/* Guess I'm BCM2711 or not. */
+	sc->sc_is2711 = ofw_bus_node_is_compatible(gpio, "brcm,bcm2711-gpio")
```

Comment:
Dropping the `? true : false` is clearer IMO
`sc->sc_is2711 = ofw_bus_node_is_compatible(gpio, "brcm,bcm2711-gpio");`